spec/features/users_signup_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+require 'rails_helper'
+
+RSpec.feature "User signs up", type: :feature do
+
+  scenario "User signs up sucessfully" do
+    visit "/signup"
+
+    fill_in "Username", :with => "AnyName"
+    fill_in "Password", :with => "password"
+    fill_in "Password confirmation", :with => "password"
+
+    click_button "Submit"
+
+    # expect(response.response_code).to eql(302) # 302 is the HTTP code for a successful signup - DOES NOT WORK
```

Comment:
I believe that response is available only in controller tests.

Which is expected I think - here we should mimic user using our webpage not responses and requests.
What you did is assertion is fine, you could also check page.curent_path :)